suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -312,3 +312,28 @@ tests:
       module: test_rbd_persistent_write_back_cache.py
       name: PWL cache creation with exclusive lock
       polarion-id: CEPH-83574719
+
+  - test:
```

Comment:
Does it not applicable to `pacific`?

Reply:
None of the persistent cache tests have been added to pacific, so I would like to take that up as part of a different PR since the scope of this PR is not to add all persistent cache tests to pacific.